config/testing/kustomization.yaml

```diff
@@ -1,7 +1,7 @@
 # Adds namespace to all resources.
 namespace: osdk-test
 
-namePrefix: osdk-
+namePrefix: smart-gateway-operator-
```

Comment:
Need this to match the main configuration otherwise the Operator will fail to load everything properly due to mismatched naming. I looked to see if there was a reasonable way to override this, but it wasn't immediately obvious. Not sure there is really an advantage to making this dynamic anyways as builds will use this value.